scrapypuppeteer/actions.py

```diff
@@ -335,3 +336,37 @@ def __init__(self, js_action: str):
 
     def payload(self):
         return self.js_action
+
+
+class Compose(PuppeteerServiceAction):
+    """
+    Compose several scrapy-puppeteer actions into one action and send it to the service.
+
+    Response for this action is PuppeteerResponse to last action in a sequence.
+
+    """
+
+    endpoint = "compose"
+
+    def __init__(self, *actions: PuppeteerServiceAction):
+        self.actions = self.__flatten(actions)
```

Comment:
is empty action list allowed?

Reply:
No! If I try to run spider with empty `Compose `, it fails when instantiating `PuppeteerRequest`, as it expects at least 1 action in `Compose`. Thanks!